images/cloud_asset_inventory/sentinel_neo4j_forwarder/Dockerfile

```diff
@@ -16,11 +16,10 @@ WORKDIR /app/
 COPY src/queries.json .
 COPY src/neo4j_connector.py /app/
 COPY src/neo4j_to_sentinel.py /app/
-COPY connector.zip /tmp/
 RUN chmod +x /app/neo4j_connector.py /app/neo4j_to_sentinel.py
 
 # Sentinel connector layer
-RUN unzip /tmp/connector.zip -d /opt && rm /tmp/connector.zip
+COPY --from=public.ecr.aws/cds-snc/aws-sentinel-connector:c1f02b3f747b80fc18ed10ab10b8c69e780b3833 /layer/. /layer
```

Comment:
This is nice - I like the trick of copying directly from the image.  I think you'll need to update the `PYTHONPATH` above though so it references this `/layer` dir:
```dockerfile
ENV PYTHONPATH "${PYTHONPATH}:/layer"
```

Reply:
One other request - could you move the commit SHA `c1f02b...` into a Docker build arg that gets passed in to the build.  That'll let us decouple the AWS sentinel connector layer from the Dockerfile and move that commit SHA up into the workflow.